tools/pd-tso-bench/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/prometheus/client_golang v1.11.1
 	github.com/tikv/pd/client v0.0.0-00010101000000-000000000000
 	go.uber.org/zap v1.24.0
+	google.golang.org/grpc v1.54.0
```

Comment:
I'm confused, https://github.com/tikv/pd/blob/master/tools/pd-tso-bench/go.mod#L12 has been updated here, why is there no "need-rebase" label? :(